modules/minimum-ibm-power-system-requirements.adoc

```diff
@@ -35,7 +35,7 @@ On your {ibm-power-name} instance, set up:
 [discrete]
 == Disk storage for the {ibm-power-title} guest virtual machines
 
-* Local storage, or storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization) or SSP (shared storage pools)
+* Local storage, or storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization), Fibre Channel or SSP (shared storage pools)
```

Comment:
```suggestion
* Local storage, or storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization), Fibre Channel, or SSP (shared storage pools)
```
https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series